pkg/component/networking/vpn/seedserver/seedserver_test.go

```diff
@@ -63,6 +63,7 @@ var _ = Describe("VpnSeedServer", func() {
 		istioNamespaceFunc = func() string { return istioNamespace }
 
 		vpaUpdateMode    = vpaautoscalingv1.UpdateModeAuto
+		vpaUpdateModeOff = vpaautoscalingv1.UpdateModeOff
```

Comment:
I personally do not feel too comfortable creating a `var` of the specific off state here.
I would rather vote for using `vpaautoscalingv1.UpdateModeOff` directly and, where required, use `ptr.To(vpaautoscalingv1.UpdateModeOff)` instead?

Reply:
Not sure why that should make a difference, but I adapted it anyway in https://github.com/gardener/gardener/pull/10486/commits/eb4df844e611cc6d7a28702b5e165bc75f89bff7.